src/doc/python_scripting/functions.rst

```diff
@@ -10110,8 +10123,8 @@ SetPrecisionType
   SetPrecisionType(typeAsString)
 
 
-typeAsInt : double
-    Precision type specified as an integer. 0 = float 1 = native 2 = double
+typeAsInt : integer
+    Precision type specified as an integer. 0 = float 1 = native 2 = double.
```

Comment:
There is a typo a few lines down on line 10136. "pecision" should be "precision".